frontend/pages/welcome.js

```diff
@@ -121,16 +123,87 @@ const Welcome = () => {
                       </Stack>
                     </AccordionPanel>
                   </AccordionItem>
+                  <AccordionItem borderWidth={0}>
+                    <h2>
+                      <AccordionButton borderWidth={0}>
+                        <Heading as="h4" size="md">
+                          How do I build my smart contract dashboard?
+                        </Heading>
+                        <AccordionIcon />
+                      </AccordionButton>
+                    </h2>
+                    <AccordionPanel pb={4} borderWidth={0}>
+                      <Stack direction="column">
+                        <chakra.span fontWeight="semibold" pl={2}>
+                          <Text fontWeight="bold" display="inline">
+                            1. Subscribe
+                          </Text>{" "}
+                          - First step is to tell us what addresses you want
+                          moonstream to track activity at. You can do this at{" "}
+                          <RouterLink passHref href="/subscriptions">
+                            <Link color="orange.500">subscriptions page</Link>
+                          </RouterLink>{" "}
+                          or simply by just clicking + on navigation bar of this
+                          page.
+                        </chakra.span>
+                        <chakra.span fontWeight="semibold" pl={2}>
+                          <Text fontWeight="bold" display="inline">
+                            2. Provide ABI to subscription
+                          </Text>{" "}
+                          - In order for us to know what data to track, you need
+                          to provide an ABI to subscription. You can upload one
+                          at{" "}
+                          <RouterLink passHref href="/subscriptions">
+                            <Link color="orange.500">subscriptions page</Link>
+                          </RouterLink>
+                          . If you are not sure what is abi ABI or which is
+                          correct for you - ask us on{" "}
+                          <Link
+                            color="orange.500"
+                            isExternal
+                            href={"https://discord.gg/K56VNUQGvA"}
+                          >
+                            Discord
+                          </Link>{" "}
+                          . <br />
+                          <i>
+                            N.B: Right now we have automatic dashboards only
```

Comment:
Remove "N.B".

Reply:
"Your dashboard will contain analytics for all the interfaces listed in your ABI. Customization is coming soon."